src/components/ebay-checkbox/README.md

```diff
@@ -0,0 +1,21 @@
+# ebay-checkbox
+
+## ebay-checkbox Usage
+
+```marko
+<ebay-checkbox/>
+```
+
+## ebay-checkbox Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`disabled` | Boolean | Yes |
+
+All the HTML attributes like `value`, `name` etc. are applied to the input tag. `class` attribute would be applied to the root tag.
```

Comment:
I think you should call out that this is different than usual case. I.e. something like:
>Note: For this component, `class` is applied to the root tag, while all other HTML attributes are applied to the `input` tag.